proposal.rst

```diff
@@ -168,82 +173,124 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+First, I have collected several resources about how to implement and improve the estimation.
+
+*** Academic
+1. (Estimating Corpus Size via Queries [CIKM'06])[http://fontoura.org/papers/cikm2006.pdf] This paper uses the basic estimator estimate the search result size. The first approach requires a uniform random sample of documents from the corpus. It avoids this notoriously difficult sample generation problem, and instead uses two fairly uncorrelated sets of terms as query pools; the accuracy of the second approach depends on the degree of correlation among the two sets of terms.
+2. (Random Sampling from a Search Engine’s Index [WWW'06]) [http://www2006.org/programme/files/pdf/3047.pdf] This paper proposes a lexicon-based technique and a random
+walk technique to avoid long documents and highly ranked documents.
+3. (Efficient Search Engine Measurements [WWW'07])[http://wwwconference.org/www2007/papers/paper753.pdf] This paper present two estimators that are able to remedy the skew introduced by approximate degrees. This estimators are based on a careful implementation of an approximate importance sampling procedure.
```

Comment:
And this paper is similarly about "external evaluation of search engines".

Reply:
Indeed, this paper is an evaluate paper. However, it gives some hints about how to remedy the bias phenomenon while estimating. In this project, I guess the core contribution is to eliminate the skewed results since the words from a query are usually correlated.